components/AnimatedRegion.js

```diff
@@ -1,45 +1,50 @@
 /* eslint-disable */
+import {Animated} from 'react-native';
 
-class AnimatedRegion extends AnimatedWithChildren {
-  // latitude: AnimatedValue;
-  // longitude: AnimatedValue;
-  // latitudeDelta: AnimatedValue;
-  // longitudeDelta: AnimatedValue;
-  // _listeners: {[key: string]: {
+const AnimatedWithChildren = Object.getPrototypeOf(Animated.ValueXY);
+
+var _uniqueId = 1;
+
+export default class AnimatedMapRegion extends AnimatedWithChildren {
+  //latitude: AnimatedValue;
+  //longitude: AnimatedValue;
+  //latitudeDelta: AnimatedValue;
+  //longitudeDelta: AnimatedValue;
+  //_listeners: {[key: string]: {
   //  latitude: string,
   //  longitude: string,
   //  latitudeDelta: string;
   //  longitudeDelta: string,
-  // }};
+  //}};
 
   constructor(valueIn) {
     super();
-    const value = valueIn || { // probably want to come up with better defaults
+    var value = valueIn || { // probably want to come up with better defaults
       latitude: 0,
       longitude: 0,
       latitudeDelta: 0,
-      longitudeDelta: 0,
+      longitudeDelta: 0
     };
-    this.latitude = value.latitude instanceof Animated
+    this.latitude = value.latitude instanceof Animated.Value
```

Comment:
Indeed, good call. I modelled it after ValueXY which also only accepts number or instances of `Animated.Value`. It didn't occur to me that you may also want to interpolate the values. I'll give it a go and see whether this is achievable.